x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/InferenceHelpers.java

```diff
@@ -20,27 +21,40 @@
 
     private InferenceHelpers() { }
 
-    public static List<ClassificationInferenceResults.TopClassEntry> topClasses(List<Double> probabilities,
-                                                                                List<String> classificationLabels,
-                                                                                int numToInclude) {
-        if (numToInclude == 0) {
-            return Collections.emptyList();
-        }
-        int[] sortedIndices = IntStream.range(0, probabilities.size())
-            .boxed()
-            .sorted(Comparator.comparing(probabilities::get).reversed())
-            .mapToInt(i -> i)
-            .toArray();
+    /**
+     * @return Tuple of the highest scored index and the top classes
+     */
+    public static Tuple<Integer, List<ClassificationInferenceResults.TopClassEntry>> topClasses(List<Double> probabilities,
```

Comment:
Is there no test for this method?

Reply:
There is coverage through functional tests.